cypress/integration/search-ui.spec.js

```diff
@@ -78,6 +81,6 @@ describe(`${playground} playground test`, () => {
 
   it('Paginate Search', () => {
     cy.get('.ais-InfiniteHits-loadMore').click()
-    cy.get(HIT_ITEM_CLASS).should('have.length', 11)
+    cy.get(HIT_ITEM_CLASS).should('have.length', 12)
```

Comment:
Why this has been changed?

Reply:
because of the disjunctive facet search accepting more hits in return 😅